src/picongpu/include/simulation_defines/param/gridConfig.param

```diff
@@ -48,7 +48,7 @@ namespace picongpu
         {32, 32}   /*z direction [negative,positive]*/
     }; //unit: number of cells
 
-    //! Define the strange of the absober for any direction
+    //! Define the strength of the absorber for any direction
```

Comment:
there are seven more `gridConfig.param` files in `examples/`. Do you mind moving that change to a separate commit and adding those?